code/__DEFINES/tools_defines.dm

```diff
@@ -13,9 +13,9 @@
 #define TOOL_DRILL "drill"
 #define TOOL_SCALPEL "scalpel"
 #define TOOL_SAW "saw"
-#define TOOL_BONESET "bonesetter"
-#define TOOL_BONEGEL "bonegel"
-#define TOOL_FIXOVEIN "fixovein"
+#define TOOL_BONESET "bone setter"
+#define TOOL_BONEGEL "bone gel"
+#define TOOL_FIXOVEIN "fix-o-vein"
```

Comment:
`TOOL_FIXOVEIN` infuriates me, i know its not the standard, but purely for ease of reading, do you think `TOOL_FIX-O-VEIN` would be okay?

Reply:
Please do not. This will make it harder to search for if specifically looking for a define. I would rather keep it in line with all the other defines that serve the same purpose.